libmscore/score.cpp

```diff
@@ -434,9 +434,10 @@ void Score::fixTicks()
                               }
                         }
                   else if (isMaster() && (s->segmentType() == Segment::Type::ChordRest)) {
-                        foreach (Element* e, s->annotations()) {
+                        for (Element* e, s->annotations()) {
```

Comment:
Syntax error, replace the comma with a colon (plus a space I think):
`for (Element* e : s->annotations()) {`